[2024-09-16T12:54:47.960+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-16T12:54:48.727+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: airbyte_sync_tables_postgres.airbyte_trigger_sync manual__2024-09-16T12:53:42.505250+00:00 [queued]>
[2024-09-16T12:54:49.240+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: airbyte_sync_tables_postgres.airbyte_trigger_sync manual__2024-09-16T12:53:42.505250+00:00 [queued]>
[2024-09-16T12:54:49.282+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-09-16T12:54:50.660+0000] {taskinstance.py:2330} INFO - Executing <Task(AirbyteTriggerSyncOperator): airbyte_trigger_sync> on 2024-09-16 12:53:42.505250+00:00
[2024-09-16T12:54:50.907+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'airbyte_sync_tables_postgres', 'airbyte_trigger_sync', 'manual__2024-09-16T12:53:42.505250+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/airbyte_sync_tables_postgres.py', '--cfg-path', '/tmp/tmpkrn4ybnk']
[2024-09-16T12:54:51.286+0000] {standard_task_runner.py:91} INFO - Job 8: Subtask airbyte_trigger_sync
[2024-09-16T12:54:51.470+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=57680) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-16T12:54:51.484+0000] {standard_task_runner.py:64} INFO - Started process 57700 to run task
[2024-09-16T12:54:53.717+0000] {task_command.py:426} INFO - Running <TaskInstance: airbyte_sync_tables_postgres.airbyte_trigger_sync manual__2024-09-16T12:53:42.505250+00:00 [running]> on host e532e2e16082
[2024-09-16T12:54:57.954+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='vizentec-engdados' AIRFLOW_CTX_DAG_ID='airbyte_sync_tables_postgres' AIRFLOW_CTX_TASK_ID='airbyte_trigger_sync' AIRFLOW_CTX_EXECUTION_DATE='2024-09-16T12:53:42.505250+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-16T12:53:42.505250+00:00'
[2024-09-16T12:54:57.966+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-16T12:54:59.191+0000] {base.py:84} INFO - Using connection ID 'airbyte_conn' for task execution.
[2024-09-16T12:55:00.886+0000] {airbyte.py:95} INFO - Job 17 was submitted to Airbyte Server
[2024-09-16T12:55:00.901+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-16T12:55:02.376+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=airbyte_sync_tables_postgres, task_id=airbyte_trigger_sync, run_id=manual__2024-09-16T12:53:42.505250+00:00, execution_date=20240916T125342, start_date=20240916T125448, end_date=20240916T125502
[2024-09-16T12:55:03.188+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-09-16T12:55:03.753+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-16T12:55:03.768+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
